README.md

```diff
@@ -1,13 +1,11 @@
 # Teehee - a modal terminal hex editor
 
-![AUR version](https://img.shields.io/aur/version/teehee)
```

Comment:
Keeping the AUR information here feels like best practice to me. Would you like to remove it?

Reply:
Forget about it, I'll revert changes in README.md
Firstly, I just not planned to push my changes to your repository so I deleted it. But then I decided to make a pull request and forgot about README.md
Yes, the AUR information is the best practice